Userland/Libraries/LibDebug/Dwarf/LineProgram.cpp

```diff
@@ -224,6 +224,15 @@ void LineProgram::handle_standard_opcode(u8 opcode)
         dbgln("SetIsa: {}", isa);
         break;
     }
+    case StandardOpcodes::FixAdvancePc: {
+        u16 delta = 0;
+        m_stream >> delta;
+#if DWARF_DEBUG
+        dbgln("FixAdvancePC by: {} to: {:p}", delta, m_address + delta);
+#endif
```

Comment:
While you're at it, could you swap these for `dbgln_if()`?
you could just cherry-pick https://github.com/alimpfard/serenity/commit/3fb80440e59f5e4c2e6be4efff36d7a3ac3d3e3f for the other cases in this file if you don't feel like doing it yourself :)

Reply:
Good idea. Done. :+1: